schemas/EGA.common-definitions.json

```diff
@@ -15,24 +15,37 @@
           "alias": {
             "type": "string",
             "title": "Alias of an object",
-            "description": "Submitter designated name (e.g. 'my_sample_J13') for the object (e.g. Sample). The name must be unique within the submission account (e.g. 'ega-box-79'), since the aliases and submission accounts are concatenated within our database to obtain the unique alias (e.g. 'ega-box-79::my_sample_J13')."
+            "description": "Submitter designated name (e.g. 'my_sample_J13') for the object (e.g. Sample). The name must be unique within the submission account (e.g. 'ega-box-79'), since the aliases and submission accounts are concatenated within our database to obtain the unique alias (e.g. 'ega-box-79::my_sample_J13').",
+            "example": "my_sample_J13"
           },
           "center_name": {
             "type": "string",
             "title": "Center name of the submitter",
-            "description": "Center name (e.g. 'EBI-TEST') associated to the submitter. In other words, it is the acronym of the submitter's account (provided by the HelpDesk team)."
+            "description": "Center name (e.g. 'EBI-TEST') associated to the submitter. In other words, it is the acronym of the submitter's account (provided by the HelpDesk team).",
+            "example": "EBI-TEST"
           },
-          "accession" : {
+          "ega_accession": {
             "type": "string",
-            "title": "Accession of the object",
-            "description": "The object accession (i.e. unique identifier) assigned by the archive (EGA). Object accessions can be found in the 'Identifiers' section of the EGA-archive website (https://ega-archive.org/metadata/how-to-use-the-api) and commonly start with EGA, followed by the distinctive letter of the object and finally the numeric ID of the instance."
+            "title": "EGA's accession of the object",
+            "description": "The object accession (i.e. unique identifier) assigned by the archive (EGA). Object accessions can be found in the 'Identifiers' section of the EGA-archive website (https://ega-archive.org/metadata/how-to-use-the-api) and commonly start with EGA, followed by the distinctive letter of the object and finally the numeric ID of the instance.",
+            "example": "EGAN00003245489"
+          },
+          "external_accessions": {
+            "type": "array",
+            "title": "External accessions array",
+            "description": "Custom attributes of an ArrayExperiment: reusable attributes to encode tag-value pairs (e.g. Tag being 'Targeted loci' and its Value '5:63256183-63258334') with optional units (e.g. 'base pairs'). Its properties are inherited from the common-definitions.json schema. #! Using an empty item list but defining the custom attributes as possible additionalItems we create the correct constraint: anything but a custom attribute is rejected, but EGA can add as many as required.",
+            "minItems": 1,
```

Comment:
Does `"minItems": 1,` mean this attribute is required? Is that too strict? i.e. will there be submissions that won't have external accessions?

Reply:
It does not, what it states is that, if given an ``eternal_accessions`` array, it needs to contain at least 1 item in the array. 
The requirements for the object's core are one of the following combinations:
````
"anyOf": [
          { "required": ["alias", "center_name"]},
          { "required": ["ega_accession"]},
          { "required": ["external_accessions"]}
        ]
````